pkgs/by-name/he/helium/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  undmg,
+  ...
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "helium";
+  version = "0.4.6.1";
+
+  src =
+    {
+      aarch64-darwin = fetchurl {
+        name = "helium_0.4.6.1_arm64-macos.dmg";
+        url = "https://github.com/imputnet/helium-macos/releases/download/0.4.6.1/helium_${finalAttrs.version}_arm64-macos.dmg";
+        hash = "sha256-K9fnxc5sDAiDj008IzwLI4BBI5mrfz3FpuECF2s+5pw=";
+      };
+      x86_64-darwin = fetchurl {
+        name = "helium_0.4.6.1_x86_64-macos.dmg";
+        url = "https://github.com/imputnet/helium-macos/releases/download/0.4.6.1/helium_${finalAttrs.version}_x86_64-macos.dmg";
```

```suggestion
  src =
    let
      inherit (finalAttrs) version;
    in
    {
      aarch64-darwin = fetchurl {
        name = "helium_${version}_arm64-macos.dmg";
        url = "https://github.com/imputnet/helium-macos/releases/download/${version}/helium_${version}_arm64-macos.dmg";
        hash = "sha256-K9fnxc5sDAiDj008IzwLI4BBI5mrfz3FpuECF2s+5pw=";
      };
      x86_64-darwin = fetchurl {
        name = "helium_${version}_x86_64-macos.dmg";
        url = "https://github.com/imputnet/helium-macos/releases/download/${version}/helium_${version}_x86_64-macos.dmg";
```
this way you, the maintainer, will have easier time to update -- it's just a matter of changing `version` and hashes

`inherit (finalAttrs) version;` is here just for the sake of saving horizontal space, cuz a string with two `finalAttrs.version` in it is a bit too long